widget/playground/src/hooks/useTheme.ts

```diff
@@ -1,24 +1,36 @@
-import { createTheme, theme } from '@rango-dev/ui';
+import {
+  createTheme,
+  darkTheme as defaultDarkTheme,
+  lightTheme as defaultLightTheme,
+} from '@rango-dev/ui';
+import { customizedThemeTokens } from '@rango-dev/widget-embedded';
 import { useEffect, useLayoutEffect, useState } from 'react';
 
 import { NOT_FOUND } from '../constants';
 import { useConfigStore } from '../store/config';
 
 export function useTheme() {
+  const [OSTheme, setOSTheme] = useState('light');
   const configTheme = useConfigStore.use.config().theme;
   const mode = configTheme?.mode;
 
-  const colors = theme.colors;
-
-  const customLightTheme = createTheme('light-theme-playground', {
-    colors,
-  });
+  const { dark, light } = customizedThemeTokens(configTheme.colors);
```

Comment:
we have a warning here: configTheme is possibly.

Reply:
fixed